pkg/proxy/iptables/proxier.go

```diff
@@ -1378,21 +1378,21 @@ func (proxier *Proxier) syncProxyRules() {
 		// Now write loadbalancing & DNAT rules.
 
 		// Firstly categorize each endpoint into three buckets:
-		//   1. all endpoints that are ready and NOT terminating
-		//   2. all endpoints that are local, ready and NOT terminating
-		//   3. all endpoints that are local, ready and terminating.
+		//   1. all endpoints that are ready and NOT terminating.
+		//   2. all endpoints that are local, ready (NOT terminating).
+		//   3. all endpoints that are local, serving and terminating.
```

Comment:
now more fixups to the iptables proxier commit, which ended up in the iptables proxier _unit tests_ commit